test/e2e/specs/onboarding/ftme__site-assembler.ts

```diff
@@ -0,0 +1,136 @@
+/**
+ * @group calypso-release
+ */
+
+import {
+	DataHelper,
+	StartSiteFlow,
+	RestAPIClient,
+	SecretsManager,
+	DomainSearchComponent,
+	SignupPickPlanPage,
+	NewSiteResponse,
+	TestAccount,
+	SiteAssemblerFlow,
+} from '@automattic/calypso-e2e';
+import { Browser, Locator, Page } from 'playwright';
+import { apiDeleteSite } from '../shared';
+
+declare const browser: Browser;
+
+describe( DataHelper.createSuiteTitle( 'Site Assembler' ), () => {
```

Comment:
**nit:non-blocking** - We're no longer using the `DataHelper.createSuiteTitle` so you are free to remove this.